### ISSUE TITLE
Argue better

### ISSUE THREAD

**Issue by Asday** [terencehuynh/alp-fail#132]
I'm not an Australian citizen, and so don't have a dog in the fight, but was made aware of the site by a colleague.

I read the first paragraph and I'm already inclined to disagree with you.  From what I understand, the bill is a bad thing for sure, but the opening paragraph leans on:

* Feelings;
* "Won't somebody please think of the children";
* And ad hominem.

It all seems very childish.

**Comment by brandonjmatthews** [comment]
I'm inclined to agree, the ALP weren't the ones who filibustered over Nauru amendments, that was the LNP. I would also suggest that this letter be directed to the LNP as well as it was originally their policy.

**Comment by terencehuynh** [comment]
With regards to the LNP stuff - I've mentioned this on Twitter, but I'll mention it here: LNP is a lost cause and ideally deserve their own letter in their own right detailing our concerns. Labor had issues with the bill, expressed those issues, and still let it through.

I am a bit weary about changing the open letter's contents because it has been viewed widely and people have signed it with that particular version of the text. If you want to make a change, I suggest you open a PR and hopefully the rest of the community agrees.

**Comment by MatthewKing** [comment]
I do agree that the LNP are a lost cause on this issue, but I also think that this open letter could benefit from addressing the fact that they need to accept a heavy share of the blame for this bill. A casual, uninformed reader could perhaps come away from reading the letter with the impression that it is mostly an "ALP fail", whereas in reality it's a "LNP and ALP fail".

Is it possible to alter the open letter's contents in a way that preserves the original intent (and thus not be seen as a "bait and switch" on the original signatories) while still addressing the fact that the LNP need to share a lot of the blame here?

Perhaps a better structure would be something like:
- Section 1. "Dear LNP" - Address the community frustration with the LNP for moving this bill.
- Section 2. "Dear ALP" - Address the community frustration with the ALP for supporting this bill (this is basically what is already present; no change really needed).
- Section 3. Arguments this bill (again, this is basically what is already present; no change really needed).

**Comment by Twistie** [comment]
I agree with @brandonjmatthews and @MatthewKing above - even if you've written off the LNP, you're allocating the blame with this page to the ALP, who are (usually) slightly less bastards and less to blame for this garbage bill. I could imagine some of my older relatives seeing this and immediately using it to help justify their support for the LNP - because the ALP killed the Australian tech industry.

**Comment by terencehuynh** [comment]
Given the focus of the letter is the ALP, I think it should stay on that (the domain is alp.fail after all).

That's not to say we shouldn't direct energy attacking the LNP too. This letter is just targeted towards the ALP.

**Comment by parsect** [comment]
indeed this is alp.fail for a reason. The way I see it is we have always known and most people do that lib/nat are morons with no clue. Until now there was still some doubt about ALP. By passing this legislation and the way they did it ALP have earned a severe bollocking and this site comes some way to expressing that. Kudos to @terencehuynh for buying the domain and putting together a slap in the face message that balances what they need to know while still being polite enough. Please don't nice it down.

**Comment by cailyoung** [comment]
Bill wouldn’t have passed the Senate without the ALP’s support, therefore they are just as responsible here as the coalition.

**Comment by Asday** [comment]
I think we instantly went off-course.

I don't know who ALP or LNP are, and I don't care.  My point is the open letter is written childishly, not that it's directed at the wrong people.

Open with what is objectively wrong with what happened, who it affects, and how it can be corrected.

**Comment by brandonjmatthews** [comment]
I only mentioned ALP and LNP at first because it tied into the inaccuracies and wording of the first paragraph. The suggestion that we direct it at LNP as well wasn't intended to take us off course and wasn't the main point of my comment. I agree it should open objectively with facts, not feelings, however as @terencehuynh said, it may be too late to change too much of the wording as it has been signed as it is.

**Comment by cailyoung** [comment]
I think given that the signatories added their signature to this letter with its current wording it would be potentially deceptive to radically alter it after the fact. 

Speaking for myself, I am personally fine with the tone. 

There are many other places that a more structured argument can be made.

**Comment by cailyoung** [comment]
Perhaps a second page with a more concrete argument and a link from the letter would achieve this without invalidating the signatures?

**Comment by parsect** [comment]
@cailyoung a link to a mainstream petition on say change.org with less confronting wording where anyone including those not on twitter or with no git fu can sign. If it is done quickly enough there will be 10,000s of signatures and it will get coverage.

**Comment by cyphar** [comment]
This is part of what I was getting at with #98. There needs to be a better argued statement of what the issues are with the bill, and why technology companies rightly are feeling very nervous about it. Unfortunately I've seen quite a lot of people spreading incorrect opinions on a law, which is quite dangerous since it can result in our legitimate concerns not being taken seriously.

Here is an example in this thread (and I've seen it said elsewhere):

> An IT worker could be faced with having to do espionage without the ability to contact a lawyer.

This is simply not true. There is common law that legal advice is generally something you have access to, even under gag orders, because of attorney-client privilege. Not only that, but the law **explicitly says** that you can contact a lawyer about a notice to get legal advice under &sect;317ZF(3)(e).

> Possibly the most important part would be producing a price charter for services to the police.

For TCNs there is an "acceptable costs negotiator" &sect;317T(12). Maybe there should be a charge for other notice types, but you are forgetting that after a capability has been provided the police could just subpoena the information (or get a new-fangled "computer access warrant" which was passed in the same law).

I also, as an Australian, don't think it's at all reasonable that large companies like Facebook get paid such exorbitant amounts of money by our government. I'd prefer my taxes go towards more useful things for our society, rather than insurance for companies like Facebook or Apple which make more than enough money already. Remember that when the government pays for something, it's not coming out of their salaries -- it's coming out of your taxes (and the other economic instruments that governments can use).

**Comment by cyphar** [comment]
As a signatory, I signed it because I agreed with the general point. If it's necessary for us to drop all the signatories in order to give a better argument (which we should be giving -- the current open letter is not good at arguing the point) then we should drop them, write a better argument and ask people to re-sign it.
